packages/components/paginator/src/page-size.scss

```diff
@@ -3,24 +3,17 @@
   color: var(--sl-color-foreground-plain);
   display: flex;
   flex-wrap: wrap;
-  font: var(--sl-text-new-body-md);
-  font-weight: var(--sl-text-typeset-font-weight-icon-regular);
-  gap: var(--sl-space-paginator-gap);
+  font: inherit;
+  gap: var(--sl-size-100);
   padding: 0;
   text-wrap: nowrap;
 }
 
 sl-select {
-  min-inline-size: var(--sl-size-paginator-select-min-width);
+  min-inline-size: var(--sl-size-1000);
 }
 
 sl-label label {
   font: inherit;
-}
-
-/** Hiding the aria-live element for the UI, but not for the screen readers. */
-#live {
-  block-size: 0;
-  inline-size: 0;
-  overflow: hidden;
+  font-weight: var(--sl-text-typeset-fontWeight-regular);
```

Comment:
Is this not the default? I only set font-weight when it should be demibold

Reply:
We are using sl-label here and label is bold, that's why I'm changing it here